share/wake/lib/system/job.wake

```diff
@@ -706,59 +706,6 @@ def runJobImp label cmd env dir stdin res usage finputs foutputs vis pers run (L
 export def runJobWith (Runner _ _ run) (Plan label cmd vis env dir stdin stdout stderr echo pers _ res _ usage finputs foutputs isatty) =
     runJobImp label cmd env dir stdin res usage finputs foutputs vis pers run echo stdout stderr isatty
```

Comment:
>   It would be a more involved migration, but do we want to take this opportunity to remove that marked-as-deprecated field?

The `score` functions look at `LocalOnly`, which deprecated its still in heavy use.
I'd like to keep this PR focused on removing the current `runJob` from the default package.

Reply:
Definitely, since they're still being used (those functions were the one place I forgot about checking).  The package split makes me all the more eager to remove the `LocalOnly` since it's not going to be used within `package wake` and having it as a core part of the tuple for an opt-in setup seems a bit ugly.  Definitely not part of this PR, though, and I'm perfectly part of it not being part of this same release.

When we do remove it, though, I wonder if we want to refactor the type to be `Plan a` so that project-specific infrastructure can attach and manipulate specific metadata.  The `package plan_scorer` would use `Plan Boolean` for local-only jobs (or the more generic `Plan ScoreInput`), for example.